src/components/Farmhand/Farmhand.js

```diff
@@ -497,6 +497,7 @@ export default class Farmhand extends FarmhandReducers {
         [toolType.WATERING_CAN]: toolLevel.DEFAULT,
       },
       useAlternateEndDayButtonPosition: false,
+      useLegacyLevelingSystem: true,
```

Comment:
It seems like this should be added to the `PERSISTED_STATE_KEYS` array in `constants.js`.